### PR TITLE
dcos-447 read json-schema from Marathon

### PR DESCRIPTION
**Pull request by tamarrow**
If old marathon version, use the cli's json-schema for now.

### CODE REVIEW

File: dcos/marathon.py [hunk 1]
```diff
@@ -556,6 +565,23 @@ def get_task(self, task_id):
...
+    def get_app_schema(self):
+        """Returns app json schema
+
+        :returns: application json schema
+        :rtype: json schema or None if endpoint doesn't exist
+        """
+
+        version = self.get_version()
+        schema_version = LooseVersion("0.9.0")
+        if version < schema_version:
```
Comment (ConnorDoyle) [inline dcos/marathon.py]: Did the integration tests run against a Marathon 0.9.0 RC yet?

Comment (tamarrow) [inline dcos/marathon.py]: I don't think so. I ran it manually on my local Marathon.

**Comment by tamarrow** [comment]
@ConnorDoyle bump

File: dcos/marathon.py (hunk 1 above)
Comment (ConnorDoyle) [inline dcos/marathon.py]: OK.  Created [DCOS-2117](https://mesosphere.atlassian.net/browse/DCOS-2117) to track the test environment updates that need to be done.

Comment (ConnorDoyle) [inline dcos/marathon.py]: Just to clarify, did you run the integration test suite against the RC?  If so :shipit:.

Comment (tamarrow) [inline dcos/marathon.py]: Yep. I'll rerun one more time before merging. thanks!

**Pull request merged**